centreon/centreon-pre-deletion.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
```

Comment:
I can't see from github, but all your scripts must have the executable bit before commiting then.